scripts/start/ensureRequirementsInstalled.R

```diff
@@ -13,8 +14,12 @@ ensureRequirementsInstalled <- function(
 ) {
   # Check if dependencies for a model run are fulfilled
   if (requireNamespace("piamenv", quietly = TRUE) && packageVersion("piamenv") >= "0.3.4") {
-    installedPackages <- piamenv::fixDeps(ask = ask)
-    piamenv::stopIfLoaded(names(installedPackages))
+    if (is.null(renv::project())) {
```

Comment:
Just an idea: You could put a `message("Checking dependencies. If this fails, use a more recent snapshot.")` or something you consider more appropriate before the call to `checkDeps` so people know what they had to do if this fails.

Reply:
Good idea, I added the message as suggested :+1: